### ISSUE TITLE
Make site-install command more verbose

### ISSUE THREAD

**Issue by gergelypap** [drush-ops/drush#4343]
When installing Drupal in the browser, the progress is transparent and the user can clearly see what extensions and configuration are being installed.

![image](https://user-images.githubusercontent.com/5883922/75981261-6e0b2500-5ee4-11ea-909a-d6d798a6a103.png)

In the CLI however, no message is logged during site install, apart from

```
[notice] Starting Drupal installation. This takes a while.
```

**Proposal:**

When using the `--verbose` flag, the command should log what is happening in the background, such as:

```
[notice] Starting Drupal installation. This takes a while.
[notice] Installed Color module.
[notice] Installed Node module.
[notice] Installed Taxonomy module.
[notice] Import configuration field.field.node.bundle.foo
# ...
```

Optionally, even more detail could be logged with the `--debug` flag:

```
[notice] Starting Drupal installation. This takes a while.
[debug] Invoking node_install()
[notice] Installed Node module.
[debug] Invoking node_modules_installed()
[notice] Importing configuration field.field.node.bundle.foo
[debug] Importing configuration field.storage.node.foo as a dependency of field.field.node.bundle.foo
# ...
```

**Reason:**

When installing a large enterprise application with hundreds of configuration files and lots of modules, the site install process can easily take up to 10 minutes. If the process fails, there is no indication as to what was the last step.

**Comment by weitzman** [comment]
I have long wanted this as well. Dupe of #4107

**Comment by weitzman** [comment]
Fixed by #4378
